etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/AddContactSectionToPatientResource.java

```diff
@@ -12,8 +12,8 @@
 import org.hl7.fhir.r4.model.Coding;
 import org.hl7.fhir.r4.model.Patient;
 
-/** Custom transformation to add a contact section to a patient resource. */
-public class addContactSectionToPatientResource implements CustomFhirTransformation {
+/** Adds mother's contact section to patient. */
+public class AddContactSectionToPatientResource implements CustomFhirTransformation {
```

Comment:
Should the `transform` method have a `try`/`catch` and thrown a `RuleExecutionException` like the newer transformations?  And then unit tests for this too.

Reply:
Same